src/pkgs.jl

```diff
@@ -381,7 +381,11 @@ function watch_files_via_dir(dirname)
         wf = watched_files[dirname]
         for (file, id) in wf.trackedfiles
             fullpath = joinpath(dirname, file)
-            if !file_exists(fullpath)
+            if isdir(fullpath)
+                println("detected dir modification: $fullpath")
```

Comment:
All looks great, but what's the motivation for the notification?

Reply:
Oops, sorry! I used this in my early tests and forgot to remove the line. I replaced it with a comment in the commit I just pushed.

Thanks for the thorough review!